pipelines/iop_saildrone/config/dataset.yaml

```diff
@@ -0,0 +1,76 @@
+attrs:
+  title: iop_saildrone
+  description: to be updated
+  location_id: cali
+  dataset_name: iop_saildrone
+  data_level: a1
+
+coords:
+  time:
+    dims: [time]
+    dtype: datetime64[ns]
+    attrs:
+      long_name: UTC time
+      standard_name: time
+      units: Seconds since 1970-01-01 00:00:00 UTC
+
+
+data_vars:
+  latitude:
+    dims: [time]
+    dtype: float
+    attrs:
+      long_name: "North latitude"
+      standard_name: "latitude"
+      description: "Recorded lattitude at the instrument location"
+      units: degree_N
+  longitude:
+    dims: [time]
+    dtype: float
+    attrs:
+      long_name: "East longitude"
+      standard_name: "longitude"
+      description: "Recorded longitude at the instrument location"
+      units: degree_E
+  air_pressure:
+    dims: [time]
+    dtype: float
+    attrs:
+      standard_name: air_pressure
+      long_name: baro_pressure_filtered
```

Comment:
We've been traditionally setting variable long names with spaces and capitalized letters, so this would be "Filtered Barometric Pressure".